_data/meltano/extractors/tap-facebook/singer-io.yml

```diff
@@ -55,15 +55,6 @@ settings:
 
       First, you will need to create a Facebook App through the Developer Portal.
 
-      ::: tip Don't feel like going through the Facebook App setup process?
-
-      <p>
-        <OAuthServiceLink provider="facebook">Connect your Facebook Ads account</OAuthServiceLink> to acquire this token right away.
```

Comment:
This functionality was from Meltano UI and is not applicable to hub.meltano.com.